tests/scripts/sendPickle.py

```diff
@@ -0,0 +1,21 @@
+import pickle
```

Comment:
This is just for dev testing/playing around, right?  can we one or two comments in this file about the function.